analyzer-comments/elixir/solution/no_integer_literal.md

```diff
@@ -0,0 +1,13 @@
+# no integer literal
+
+Elixir has a [special syntax](https://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html#unicode-and-code-points) to write integers that correspond to [Unicode Code Points](https://en.wikipedia.org/wiki/Code_point): you can use a `?` in front of the character literal to reveal its code point.
+
+For example,
+
+```elixir
+?a === 97
+?ß === 223
+?互 === 20114
```

Comment:
Haha, that's my favorite kanji. お互い means reciprocal, or each other, and I find the kanji aesthetically pleasing because it looks like two people locking arms from above, kinda like that:
![Screen Shot 2021-10-14 at 9 34 19](https://user-images.githubusercontent.com/16929078/137231072-b3a11045-3a1b-45bf-8b29-1d56202ee307.png)